lte/gateway/c/oai/tasks/amf/amf_Security_Mode.cpp

```diff
@@ -68,7 +68,7 @@ int amf_handle_security_complete_response(
      * TODO:Stop timer T3560 This to be taken care in upcoming PR
      */
 
-    stop_timer(&amf_app_task_zmq_ctx, smc_proc->T3560.id);
+    amf_app_stop_timer(smc_proc->T3560.id);
     OAILOG_DEBUG(LOG_AMF_APP, "Timer: After stopping SMC MODE timer \n");
```

Comment:
let us not indicate with informal naming for the timer such as "SMC MODE timer" or something like. let us use an uniform naming T3xxx timer.